src/services/updateCourtCase.ts

```diff
@@ -0,0 +1,25 @@
+import { DataSource, UpdateResult } from "typeorm/"
+import CourtCase from "./entities/CourtCase"
+
+const updateCourtCaseUpdatedHO = async (
```

Comment:
The next function has the same name, but this function name doesn't correspond with the file name `updateCourtCase.ts`, did you forget to rename it?